frontend/src/components/UserDetail/userDetail.module.css

```diff
@@ -0,0 +1,60 @@
+.userDetail {
+  width: 90%;
+  display: inline-block;
+  border-radius: 8px;
+  background-color: white;
+  box-shadow: 0px 4px 35px rgba(0, 0, 0, 0.1);
+  margin-left: 5%;
+}
+
+.imgContainer {
+  margin: 30px 0px 0px 20px;
+  display: inline-block;
+  width: 150px;
+  height: 150px;
+}
+
+.profilePic {
+  background-color: #dddddd;
+  width: 120px;
+  height: 120px;
```

Comment:
Avoid using pixels in CSS whenever possible. Go through and try to make styling as responsive as you can. Test on multiple screen sizes using your browser's responsive design dev tools -- should ensure the styling looks fine on screens the size of an iPad and larger (dispatchers will not be using this application on mobile screens, so don't worry about those).